packages/frontend-2/components/settings/workspaces/security/sso/Wrapper.vue

```diff
@@ -25,22 +25,29 @@
         <FormButton v-else @click="goToBilling">Upgrade to Plus</FormButton>
       </div>
 
+      <div
```

Comment:
You can use `CommonCard`, if it's just for now for testing purposes no need to change it